userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -133,13 +148,17 @@ Invoice Status:: The current state of an invoice, which can be in a `DRAFT`, `CO
 
 == J
 
+Janitor:: An internal Kill Bill service that helps ensure that payment transactions end up in a terminal state.
+
 == K
 
 [[kaui]]
 Kaui:: An acronym for Kill (Bill) Admin User Interface, this browser-based application lets back office staff (in Finance, Marketing, Development, etc.) perform Kill Bill-related tasks, such as creating customer accounts, subscriptions, adding plugins, and so forth.
 
 Kill Bill Administrator (or Kill Bill Admin):: The person responsible for installing or configuring Kill Bill.
 
+Kill Bill Package Manager (or kpm) :: A command line tool that facilitates the installation of Kill Bill, its plugins and Kaui. It also provides utility helpers useful for the day-to-day management of a production system.
```

Comment:
```suggestion
Kill Bill Package Manager (kpm or KPM):: A command line tool that facilitates the installation of Kill Bill, its plugins, and Kaui. It also provides utility helpers useful for the day-to-day management of a production system.
```

Reply:
Yes, I know we don't use the Oxford comma elsewhere in the docs, but Rome wasn't built in a day! :-)